app/templates/components/public/featured-speaker-list.hbs

```diff
@@ -0,0 +1,8 @@
+<h3>{{t 'Featured Speakers'}}</h3>
+<div class="speaker-list ui grid">
+  {{#each speakers as |speaker|}}
+    {{#if speaker.featured}}
```

Comment:
As you have mentioned here instead of looping through all speakers we will compute featuredSpeakers and loop over it. That is the reason for having two separate arrays.